Mobile Device Mode Destination Filters Beta (Swift, Kotlin, React Native 2.0) [DOC-589] (#3598)
segmentio/segment-docs: src/connections/sources/catalog/libraries/mobile/swift-ios/index.md

Though you can add plugins anywhere in your code, it's best to implement your plugin when you configure the client.

## Destination filters
> info ""
> Destination filters are only available to Business Tier customers.

> You can set up [destination filters](docs/connections/destinations/destination-filters/) on your mobile device-mode destinations by using Analytics Swift.

Reviewer: Same as above regarding active voice

> You can set up [destination filters](docs/connections/destinations/destination-filters/) on your mobile device-mode destinations by using Analytics Swift.

Author: ```suggestion
Use Analytics Swift to set up [destination filters](docs/connections/destinations/destination-filters/) on your mobile device-mode destinations.
```

To get started with destination filters using Swift:

> 1. Add the plugin dependency to your application by adding the Swift package `git@github.com:segmentio/DestinationFilters-Swift.git` as a dependency through either of these 2 options:

Author: ```suggestion
1. Add the Swift package `git@github.com:segmentio/DestinationFilters-Swift.git` as a dependency through either of these 2 options:
```

> 1. Add the plugin dependency to your application by adding the Swift package `git@github.com:segmentio/DestinationFilters-Swift.git` as a dependency through either of these 2 options:

Reviewer: This reads a bit weird to me, I think because it mentions adding dependencies twice, even though just one thing needs to be added to the application.

    1. Your package.swift file
    2. Xcode
        1. Xcode 12: **File > Swift Packages > Add Package Dependency**
        2. Xcode 13: **File > Add Packages...**

>     After installing the package, you can reference the Destination Filters plugin by importing the package with `import DestinationFilters_Swift`.

Reviewer: ```suggestion
    After you install the package, import the package with `import DestinationFilters_Swift` to reference the Destination Filters plugin.
```

Active voice again
